nixos/modules/services/misc/jackett.nix

```diff
@@ -60,6 +60,56 @@ in
         Group = cfg.group;
         ExecStart = "${cfg.package}/bin/Jackett --NoUpdates --DataFolder '${cfg.dataDir}'";
         Restart = "on-failure";
+        
+        # Sandboxing
+        CapabilityBoundingSet = [
+          "CAP_NET_BIND_SERVICE"
+        ];
+        ExecPaths = [
+          "${builtins.storeDir}"
+        ];
+        LockPersonality = true;
+        NoExecPaths = [
+          "/"
+        ];
+        NoNewPrivileges = true;
+        PrivateDevices = true;
+        PrivateMounts = true;
+        PrivateTmp = true;
+        PrivateUsers = true;
+        ProtectClock = true;
+        ProtectControlGroups = true;
+        ProtectHostname = true;
+        ProtectKernelLogs = true;
+        ProtectKernelModules = true;
+        ProtectKernelTunables = true;
+        ProtectProc = "invisible";
+        ProtectSystem = "strict";
+        ReadWritePaths = [
+          "${config.users.users.${cfg.user}.home}"
+          "/tmp"
```

Comment:
Libraries of the .NET framework uses this for creating files (I don't know why, but the app does not start if I make it read-only).

Reply:
`PrivateTmp = true` would be better here, I'd say.

Reply:
That's already there (see above). But `ProtectSystem = "strict"` makes the whole filesystem read-only, that's why I added `/tmp` here (Though not sure if `PrivateTmp` implicitly overrides `ProtectSystem` or not, I will check.).

Reply:
Looks like `/tmp` is redundant, when `PrivateTmp` is set. I'll update the PR.